kalite/topic_tools/__init__.py

```diff
@@ -151,7 +152,7 @@ def get_exercise_cache(force=False, language=settings.LANGUAGE_CODE):
                 items = []
                 for item in exercise.get("all_assessment_items","[]"):
                     item = json.loads(item)
-                    if assessmentitems.get(item.get("id")):
+                    if main_models.AssessmentItem.objects.filter(id=item.get("id")).exists():
```

Comment:
Could we use get_assessment_item_data function here so that everything flows through one function?